perllib/FixMyStreet/Script/ArchiveOldEnquiries.pm

```diff
@@ -138,9 +141,15 @@ sub close_with_emails {
         rows => $opts->{limit},
     })->count;
 
-    printf("%d users will receive closure emails about %d reports which will be closed.\n", $user_count, $problem_count);
-
-    if ( $opts->{commit} ) {
+    if ( $opts->{show_emails} ) {
+        printf("%d users would have received closure emails about %d reports. These will not be closed.\n", $user_count, $problem_count);
```

Comment:
This if block and the next elsif block are identical apart from this line of print output.

Reply:
The script generally has the same output whether commit is used or not, but without commit it shows a message at the top saying nothing will happen. This changes now changes that, in that running without commit or show_emails it wouldn't print the "will receive closure emails" line at all - so you won't get the information you would previously. I don't think any changes need to be made to the `close_with_emails` function.